packages/api/cms-api/src/dam/dam.module.ts

```diff
@@ -46,7 +46,7 @@ export class DamModule {
 
         return {
             module: DamModule,
-            imports: [MikroOrmModule.forFeature([File, Folder, FileImage, ImageCropArea]), BlobStorageModule],
+            imports: [MikroOrmModule.forFeature([File, Folder, FileImage, ImageCropArea]), BlobStorageModule, BlocksModule],
```

Comment:
why do you need this import?

Reply:
answer: for `this.blockIndexService.getDependentsByTargetEntityNameAndTargetId`

not good to have a dependency on BlocksModule, maybe dependencies should be in it's own module?

Reply:
I extracted the `DiscoverService` and `BlockIndexService` to a `DependenciesModule`.